dataactbroker/handlers/fileHandler.py

```diff
@@ -195,9 +195,12 @@ def checkSubmissionPermission(self,submission):
         """
         userId = LoginSession.getName(session)
         user = self.interfaces.userDb.getUserByUID(userId)
-        # Check that user has permission to see this submission, user must be within the agency of the submission
-        if(submission.cgac_code != user.cgac_code and submission.user_id != user.user_id):
-            raise ResponseException("User does not have permission to view that submission",StatusCode.CLIENT_ERROR)
+        # Check that user has permission to see this submission, user must be within the agency of the submission, or be
+        # the original user, or be in the 'SYS' agency
+        if(submission.cgac_code != user.cgac_code and submission.user_id != user.user_id
+           and user.cgac_code != "SYS"):
```

Comment:
I didn't think of this when converting to cgac_code but should those comparisons be case sensitive?